--- orig/html/mainfile.php	2003-03-28 00:26:32.000000000 -0600
+++ html/mainfile.php	2003-04-15 17:57:53.000000000 -0500
@@ -92,7 +92,7 @@
 $startdate = $row[startdate];
 $adminmail = $row[adminmail];
 $anonpost = $row[anonpost];
-$Default_Theme = $row[Default_Theme];
+$Default_Theme = $row[default_theme];
 $foot1 = $row[foot1];
 $foot2 = $row[foot2];
 $foot3 = $row[foot3];
@@ -136,10 +136,10 @@
 $admingraphic = $row[admingraphic];
 $httpref = $row[httpref];
 $httprefmax = $row[httprefmax];
-$CensorMode = $row[CensorMode];
-$CensorReplace = $row[CensorReplace];
+$CensorMode = $row[censormode];
+$CensorReplace = $row[censorreplace];
 $copyright = $row[copyright];
-$Version_Num = $row[Version_Num];
+$Version_Num = $row[version_num];
 $domain = eregi_replace("http://", "", $nukeurl);
 $tipath = "images/topics/";
 $mtime = microtime();
